include/ozo/transaction.h

```diff
@@ -1,22 +1,38 @@
 #pragma once
 
+#include <ozo/core/options.h>
+#include <ozo/detail/begin_statement_builder.h>
 #include <ozo/impl/async_start_transaction.h>
 #include <ozo/impl/async_end_transaction.h>
+#include <ozo/transaction_status.h>
 
 namespace ozo {
 
-template <typename Initiator>
-struct begin_op : base_async_operation <begin_op, Initiator> {
+template <typename Initiator, typename StatementBuilder, typename Options = decltype(make_options())>
```

Comment:
Does it really need to point `StatementBuilder` as a template parameter? Does it really need to be customized? I mean what is the customization point it provides?

Reply:
The original intention was to have it customizable so you could, in theory, roll your own statement builder, e.g. when a newer PostgreSQL version, extension or fork would add additional options for the begin statement.
That being said, this is indeed very unlikely, so I did remove it. If anyone ever needs it, it's easy enough for them to put it back in.